csharp/Platform.Ranges/Range[T].cs

```diff
@@ -58,89 +70,144 @@ public Range(T minimum, T maximum)
+        /// <para>Экземпляр <see cref="Range{T}"/> для сравнения с текущим экземпляром.</para>
+        /// </param>
+        /// <returns>
+        /// <para>A <see cref="Boolean"/> value that determines whether the current <see cref="Range{T}"/> instance and <paramref name="other"/> are equal.</para>
+        /// <para>Значение типа <see cref="Boolean"/>, определяющее, равны ли текущий экземпляр <see cref="Range{T}"/> и <paramref name="other"/>.</para>
+        /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public bool Equals(Range<T> other) => _equalityComparer.Equals(Minimum, other.Minimum) && _equalityComparer.Equals(Maximum, other.Maximum);
 
         /// <summary>
-        /// <para>Creates a new <see cref="ValueTuple{T,T}"/> struct initialized with <see cref="Range{T}.Minimum"/> as <see cref="ValueTuple{T,T}.Item1"/> and <see cref="Range{T}.Maximum"/> as <see cref="ValueTuple{T,T}.Item2"/>.</para>
-        /// <para>Создает новую структуру <see cref="ValueTuple{T,T}"/>, инициализированную с помощью <see cref="Range{T}.Minimum"/> как <see cref="ValueTuple{T,T}.Item1"/> и <see cref="Range{T}.Maximum"/> как <see cref="ValueTuple{T,T}.Item2"/>.</para>
+        /// <para>Initializes a new <see cref="ValueTuple{T,T}"/> struct with the with the <see cref="Minimum"/> and <see cref="Maximum"/> values of the <paramref name="range"/>.</para>
+        /// <para>Инициализирует новую структуру <see cref="ValueTuple{T,T}"/>, с значениями <see cref="Minimum"/> и <see cref="Maximum"/> объекта <paramref name="range"/>.</para>
         /// </summary>
-        /// <param name="range"><para>The range of <typeparamref name="T"/>.</para><para>Диапазон значений <typeparamref name="T"/>.</para></param>
+        /// <param name="range">
+        /// <para>A <see cref="Range{T}"/> instance to initialize a tuple with the <see cref="Minimum"/> and <see cref="Maximum"/> values.</para>
+        /// <para>Экземпляр <see cref="Range{T}"/> для инициализации кортежа с значениями <see cref="Minimum"/> и <see cref="Maximum"/>.</para>
+        /// </param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static implicit operator ValueTuple<T, T>(Range<T> range) => (range.Minimum, range.Maximum);
 
         /// <summary>
-        /// <para>Creates a new <see cref="Range{T}"/> struct initialized with <see cref="ValueTuple{T,T}.Item1"/> as <see cref="Range{T}.Minimum"/> and <see cref="ValueTuple{T,T}.Item2"/> as <see cref="Range{T}.Maximum"/>.</para>
-        /// <para>Создает новую структуру <see cref="Range{T}"/>, инициализированную с помощью <see cref="ValueTuple{T,T}.Item1"/> как <see cref="Range{T}.Minimum"/> и <see cref="ValueTuple{T,T}.Item2"/> как <see cref="Range{T}.Maximum"/>.</para>
+        /// <para>Initializes a new <see cref="Range{T}"/> struct with the <see cref="Minimum"/> and <see cref="Maximum"/> values from the <paramref name="tuple"/> in the according order.</para>
```

Comment:
```suggestion
        /// <para>Initializes a new <see cref="Range{T}"/> struct using the first tuple value as <see cref="Minimum"/> and second tuple value as <see cref="Maximum"/> from the <paramref name="tuple"/>.</para>
```

Reply:
@Konard There is only one tuple. 
```suggestion
        /// <para>Initializes a new <see cref="Range{T}"/> struct using the first tuple item as the <see cref="Minimum"/> and second tuple item as the <see cref="Maximum"/> from the <paramref name="tuple"/>.</para>
```
or
```suggestion
        /// <para>Initializes a new <see cref="Range{T}"/> struct using the first tuple item value as the <see cref="Minimum"/> and second tuple item value as the <see cref="Maximum"/> from the <paramref name="tuple"/>.</para>
```

Reply:
```suggestion
        /// <para>Initializes a new <see cref="Range{T}"/> struct using items of the <paramref name="tuple"/>: the first item as the <see cref="Minimum"/> and the second item as the <see cref="Maximum"/>.</para>
```